### DISCUSSION TITLE
Error ccess to fetch at 'https://explorer-api.walletconnect.com/w3m/v1/getDesktopListings CORS 400

### DISCUSSION THREAD

**Discussion by valeryV99** [WalletConnect/walletconnect-monorepo#2971] [Developer Support] [answered]
<img width="587" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/29901430/d553531f-730e-449a-ab4e-8a6b218bc25f">

**Comment by Cali93** [comment, marked as answer]
Thanks for reporting this issue. We are aware of it and the fix should be released by tomorrow.
It's very likely that you are getting this error because your `projectId` is invalid.

**Reply by valeryV99** [reply]
but it works in localhost environment
